### PR TITLE
fix: (datastore) add imports to snippets

### PR DESCRIPTION
**Pull request by daniel-sanche**
Some of the datastore snippets are missing required imports. This change pulls the necessary imports into each sample in snippets.py

### CODE REVIEW

File: datastore/cloud-client/snippets.py [hunk 1]
```diff
@@ -887,6 +913,9 @@ def property_by_kind_run_query(client):
 
 def regional_endpoint():
     # [START datastore_regional_endpoints]
+    from google.cloud import datastore
```
Comment (gkevinzheng) [inline datastore/cloud-client/snippets.py:916]: Why is this the only sample that has this import?

Comment (daniel-sanche) [reply datastore/cloud-client/snippets.py:916]: All the other ones also include [the preamble](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/ec9128da73d7832cb970cd0442562a54be1a1ad1/datastore/cloud-client/snippets.py#L17), which imports datastore and creates the client

**Review by gkevinzheng — APPROVED** [review]
LGTM

**Review by iennae — APPROVED** [review]
recognizing that this is a pre-existing issue where there are a bunch of samples in one file rather than one sample per file, (hopefully this can get refactored and fixed in the future) but also wanting to help unblock getting this fixed so that samples are copy pastable where they are showing up in c.g.c approving this.

**Pull request merged**